lib/src/native/realm_core.dart

```diff
@@ -384,6 +384,14 @@ class _RealmCore {
     });
   }
 
+  void realmListRemoveAll(RealmList list) {
+    _realmLib.invokeGetBool(
+        () => _realmLib.realm_list_remove_all(list.handle._pointer));
+  }
+
+  void realmResultsDeleteAll(RealmResults results) {
+    _realmLib.invokeGetBool(() => _realmLib.realm_results_delete_all(results.handle._pointer));
+  }
```

Comment:
Are these doing the same thing? I.e. deleting the object from the Realm? If so, should we name them consistently?

Reply:
Yes, you are right. I named them similar to the API, but we will discuss these names, maybe they have to be consistent.

Reply:
Renamed to realmListDeleteAll and realmResultsDeleteAll.